packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import { FieldMessage } from "@kaizen/draft-form"
+import {
+  DateRangeValidationMessageType,
+  DateRangeValidationStatus,
+  isDateRangeMessageType,
+} from "../../../types"
+import styles from "./DateRangeValidationMessage.module.scss"
+
+export type DateRangeValidationMessageProps = {
+  /**
+   * Updates the styling of the validation FieldMessage
+   */
+  status?: DateRangeValidationStatus
+  /**
+   * A descriptive message for `status` states
+   */
+  validationMessage?: DateRangeValidationMessageType
+  errorMessageId?: string
+  isReversed?: boolean
+}
+
+export const DateRangeValidationMessage = ({
+  status,
+  validationMessage,
+  errorMessageId,
+  isReversed,
+}: DateRangeValidationMessageProps): JSX.Element | null => {
```

Comment:
```suggestion
}: DateRangeValidationMessageProps): JSX.Element => {
```

It looks like you'll always be returning a JSX.Element :)

Reply:
Alternatively, you could also add something like this to the top:

```
if (!validationMessage) return null
```